packages/lib/src/allTimeZones.ts

```diff
@@ -0,0 +1,110 @@
+const allTimeZones = [
```

Comment:
Can you add the a GMT prefix to all labels so that is looks like this:

![CleanShot 2024-12-20 at 09 59 15@2x](https://github.com/user-attachments/assets/3ee722e9-64b4-43ee-9029-fa2ad173d56c)